client/state/sharing/services/test/selectors.js

```diff
@@ -8,6 +8,8 @@ import { expect } from 'chai';
  */
```

Comment:
```
wp-calypso (update/7919-use-keyring-services) $ ./node_modules/.bin/eslint client/state/sharing/services/test/selectors.js

/Users/andrew/Documents/Code/wp-calypso/client/state/sharing/services/test/selectors.js
  101:7   warning  'state' is never reassigned. Use 'const' instead  prefer-const
  146:34  warning  A space is required after '['                     computed-property-spacing
  146:42  warning  A space is required before ']'                    computed-property-spacing
  148:34  warning  A space is required after '['                     computed-property-spacing
  148:42  warning  A space is required before ']'                    computed-property-spacing
  154:34  warning  A space is required after '['                     computed-property-spacing
  154:42  warning  A space is required before ']'                    computed-property-spacing
  156:34  warning  A space is required after '['                     computed-property-spacing
  156:42  warning  A space is required before ']'                    computed-property-spacing
  175:21  warning  A space is required after '['                     computed-property-spacing
  175:29  warning  A space is required before ']'                    computed-property-spacing
  177:21  warning  A space is required after '['                     computed-property-spacing
  177:29  warning  A space is required before ']'                    computed-property-spacing

✖ 13 problems (0 errors, 13 warnings)
```

I'd recommend installing an ESLint plugin for your editor so these issues are more apparent.

See: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#eslint